flutter/example/pubspec.yaml

```diff
@@ -21,9 +21,4 @@ dependency_overrides:
     path: ../
 
 flutter:
-  uses-material-design: true
-  plugin:
```

Comment:
I'm afraid that this would remove the `web` badge compatibility from pub.dev
how can we check that?

see an example:
https://github.com/flutter/plugins/blob/master/packages/url_launcher/url_launcher_web/pubspec.yaml

do we need to add at least an empty plugin or not?

Reply:
This is the pubspec file of the example not of Sentry itself. To support web, we don't need to do something special. Also  pana in CI would complain if it's not compatible with web anymore. We also have different imports for web and io where needed.

Reply:
url_launcher has that because it is a federated plugin which Sentry is not.

Reply:
The web build in CI is also green :D

Reply:
yep, you're totally right, I need a coffee :)